website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -136,6 +136,7 @@ security fixes.
 
 | `consul-dataplane` Version Range | Associated Consul Enterprise version     | Contained Envoy Binary Version |
 | -------------------------------- | ---------------------------------------- | ------------------------------ |
+| 1.6.0 - 1.6.latest               | 1.20.x Ent                               | Envoy 1.31.x                   |
```

Comment:
We should probably correct the second column below as well; it looks like starting w/ 1.2.x we conflated "Ent" w/ "Ent LTS"

Reply:
True, it's easy to get "active versions of Consul" confused with "LTS versions of Consul"

Reply:
Done